aiosmtpd/smtp.py

```diff
@@ -149,6 +152,8 @@ def _handle_client(self):
                 continue
             yield from method(arg)
 
+            yield  # give a change to switch to another coroutine
```

Comment:
Small coding style nits: I prefer to use non-hanging full sentences for comments (e.g. starts with a capital letter, ends in a period).  Also, no extraneous vertical whitespace.  Could you move the comment up to the preceding blank line and make it a full sentence?